### DISCUSSION TITLE
My "compute_roc_auc" does not work in V0.5.0

### DISCUSSION THREAD

**Discussion by naruetep** [Project-MONAI/MONAI#2059] [Q&A]
from monai.metrics import compute_roc_auc  -> Ok 

but 
compute_roc_auc(y_pred, y, to_onehot_y=True, softmax=True) ; use to work before but failed in  V0.5.0

Error message:
TypeError: compute_roc_auc() got an unexpected keyword argument 'to_onehot_y'

**Comment by Nic-Ma** [comment]
Hi @naruetep , `to_onehot_y` arg is dropped in MONAI v0.5.0, please use post transform to do OneHot:
https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/post/array.py#L133

Thanks.
